EnglishFilter/sections/whitelist.txt

```diff
@@ -4,6 +4,9 @@
 !#########################################
 ! START: Temporary
 !
+! 
+! the number indicator of rankings blocked by /img/ad/*
+@@||ascii.jp/img/ad/yayoi/rank_$image,~third-party
```

Comment:
what about to report to EL?

Reply:
Ofc fine, I'll do that but it may take time for them to add the rule.

Reply:
PRed. https://github.com/easylist/easylist/pull/6197